ADAL/tests/integration/ADAuthorityValidationIntegrationTests.m

```diff
@@ -104,16 +99,18 @@ - (void)testAadNormalFlow
     
     [ADTestURLSession addResponse:[ADTestURLResponse responseValidAuthority:authority]];
     
+    XCTestExpectation* expectation = [self expectationWithDescription:@"Validate valid authority."];
     [authorityValidation validateAuthority:requestParams
                            completionBlock:^(BOOL validated, ADAuthenticationError * error)
      {
          XCTAssertTrue(validated);
          XCTAssertNil(error);
          
-         TEST_SIGNAL;
+         [expectation fulfill];
      }];
     
-    TEST_WAIT;
+    [self waitForExpectationsWithTimeout:1 handler:nil];
```

Comment:
nit: Also, if it's the same number everywhere, maybe we can define it somewhere as a constant?